### PR TITLE
Fix update/install commands in Docker image test code.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

I’m genuinely not sure how this was working previously...

##### Test Plan

Docker CI passes on this PR.

### CODE REVIEW

**Comment by ilyam8** [comment]
Yes, installing `netcat` never worked 😄 

---

@Ferroin I think either `nc` or `netcat` tool is available and we don't need to install it - the issues in https://github.com/netdata/netdata/runs/5514817602?check_suite_focus=true is 

> Waiting for netdata on localhost:19999 ... Timed out!

not

> Unable to find a usable netcat command.

**Review by ilyam8 — APPROVED** [review]

File: .github/scripts/docker-test.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ wait_for() {
   printf "OK\n"
 }
 
-apt-get update && apt-get upgrade -y && apt get install -y netcat
+sudo apt-get update && sudo apt-get upgrade -y && sudo apt-get install -y netcat
```
Comment (ilyam8) [inline .github/scripts/docker-test.sh:36]: It appears we don't need them.

Comment (ilyam8) [inline .github/scripts/docker-test.sh:36]:
```suggestion
```
https://github.com/netdata/netdata/runs/5514863217?check_suite_focus=true

> 101 upgraded, 0 newly installed, 0 to remove and 8 not upgraded.
54 standard security updates
> Need to get 535 MB of archives.
> After this operation, 21.9 MB of additional disk space will be used.

> **netcat is already the newest version (1.206-1ubuntu1).**

**Review by ilyam8 — CHANGES_REQUESTED** [review]

File: .github/scripts/docker-test.sh (hunk 1 above)
Comment (Ferroin) [reply .github/scripts/docker-test.sh:36]: We should not be relying on `netcat` being installed here (in the past it has not been by default, and I would not bet on it continuing to be the default), and if we0re installing packages we need to pull in any possible updates for consistency reasons (`APT can behave rather strangely if you try to install packages when you have pending updates).

Comment (Ferroin) [reply .github/scripts/docker-test.sh]: Done.

**Review by ilyam8 — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Review by maneamarius — APPROVED** [review]

**Pull request merged**